rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -0,0 +1,233 @@
+#ifdef QT_UI
```

Comment:
? qmake just generates Makefile files for whatever configuration/compiler you've specified.
include guards/pragmas has nothing to do with qmake/cmake/MSbuild and etc.